include/podio/CollectionBase.h

```diff
@@ -61,6 +61,8 @@ class CollectionBase {
   virtual std::string getValueTypeName() const = 0;
   /// fully qualified type name of stored POD elements - with namespace
   virtual std::string getDataTypeName() const = 0;
+  /// schema version of the collection
+  virtual std::string getSchemaVersion() const = 0;
```

Comment:
Yes, we can easily switch to an int. I put that flexibility in because of now being able to have derived data models. And that kills the linearity in version numbers if both PODIO and other model change. 
Just a policy decision.